dbscripts/update-001.sql

```diff
@@ -1,3 +1,4 @@
 alter table user_removals add column ldap_username varchar(255);
 alter table user_removals add column github_username varchar(255);
 update user_removals set ldap_username = username;
+ALTER TABLE REPOSITORY_FORKS  ADD DELETED TIMESTAMP AFTER CREATED;
```

Comment:
Put this into separate file please, update-002.sql, so it can be executed separately.